src/main/resources/application-bootRun.yml

```diff
@@ -32,4 +32,8 @@ servicediscovery:
   enabled: true
 
```

Comment:
Since you've put some stuff in debug logging, can you enable it for bootrun?

Reply:
nah, it's going to be temporary, not structural ?